### ISSUE TITLE
Namespace import disappeared in output

### ISSUE THREAD

**Issue by davidlj95** [timocov/dts-bundle-generator#319]
**Bug report**

Namespace import disappeared starting from version `9.4.0`

My use case is generating a bundle from Typescript definition files emitted after building an Angular library. But could reproduce it manually. See:

https://github.com/davidlj95/dts-bundle-generator-missing-import-repro

Fortunately, the check step found out the output was missing that import

**Input code**

```ts
import { ModuleWithProviders } from '@angular/core';
import * as i0 from "@angular/core";

export declare class SampleModule {
    static forRoot(): ModuleWithProviders<unknown>;
    static ɵfac: i0.ɵɵFactoryDeclaration<unknown, never>;
    static ɵmod: i0.ɵɵNgModuleDeclaration<unknown, never, never, never>;
    static ɵinj: i0.ɵɵInjectorDeclaration<unknown>;
}
```

**Expected output**

```ts
// Generated by dts-bundle-generator v9.4.0
import { ModuleWithProviders } from '@angular/core';
import * as i0 from "@angular/core";

export declare class SampleModule {
	static forRoot(): ModuleWithProviders<unknown>;
	static ɵfac: i0.ɵɵFactoryDeclaration<unknown, never>;
	static ɵmod: i0.ɵɵNgModuleDeclaration<unknown, never, never, never>;
	static ɵinj: i0.ɵɵInjectorDeclaration<unknown>;
}

export {};

```

**Actual output**

```ts
// Generated by dts-bundle-generator v9.4.0

import { ModuleWithProviders } from '@angular/core';

export declare class SampleModule {
	static forRoot(): ModuleWithProviders<unknown>;
	static ɵfac: i0.ɵɵFactoryDeclaration<unknown, never>;
	static ɵmod: i0.ɵɵNgModuleDeclaration<unknown, never, never, never>;
	static ɵinj: i0.ɵɵInjectorDeclaration<unknown>;
}

export {};
```

**Additional context**
Add any other context about the problem here (CLI options, etc)

See minimal repro repo
https://github.com/davidlj95/dts-bundle-generator-missing-import-repro

**Comment by timocov** [comment]
Broken in eb862bfdf019bd0a782b60090a87f6077505ec5c

**Comment by timocov** [comment]
Thanks for reporting the issue! The fix has been published in [v9.5.1](https://github.com/timocov/dts-bundle-generator/releases/tag/v9.5.1).

**Comment by davidlj95** [comment]
Awesome 💪  Thanks to you for this great project ❤️
